bika/lims/browser/worksheet/templates/add_analyses.pt

```diff
@@ -1,133 +1,137 @@
+                <option
+                  tal:attributes="value python:service;
+                                  selected python: view.request.form.get(form_id+'_Title', '') == service and 'selected' or ''"
+                  tal:content="python:services_list.getValue(service)"/>
+              </tal:options>
+            </select>
+          </div>
 
-			<label tal:attributes="for string:${form_id}_FilterByClient"
-				i18n:translate="">Client</label>
-			<select id="ClientSelector"
-					tal:attributes="name string:${form_id}_FilterByClient"
-					tal:define="clients_list view/getClients"
-					class="listing-filter">
-				<option value="any" i18n:translate="">Any</option>
-				<tal:options repeat="client clients_list">
-					<option
-						tal:attributes="
-							value python:client;
-							selected python: view.request.form.get(form_id+'_getClientTitle', '') == client and 'selected' or ''"
-						tal:content="python:clients_list.getValue(client)"/>
-				</tal:options>
-			</select>&nbsp;
-			<a tal:attributes="href view/view_url" i18n:translate=""
-				class="ws-analyses-search-button">Search</a>
-		</fieldset>
+          <div class="field">
+            <label tal:attributes="for string:${form_id}_FilterByClient"
+                   i18n:translate="">Client</label>
+            <select id="ClientSelector"
+                    tal:attributes="name string:${form_id}_FilterByClient"
+                    tal:define="clients_list view/getClients"
+                    class="listing-filter">
+              <option value="any" i18n:translate="">Any</option>
+              <tal:options repeat="client clients_list">
+                <option
+                  tal:attributes="value python:client;
+                                  selected python: view.request.form.get(form_id+'_getClientTitle', '') == client and 'selected' or ''"
+                  tal:content="python:clients_list.getValue(client)"/>
+              </tal:options>
+            </select>
+          </div>
 
-	</tal:searchform>
-	<fieldset/>
+          <!-- Search Button -->
+          <a tal:attributes="href view/view_url"
+             class="ws-analyses-search-button"
+             i18n:translate="">Search</a>
 
-	<div id="folderlisting-main-table"
-		tal:content="structure view/contents_table"/>
+        </fieldset>
+      </tal:searchform>
 
-	<tal:remarks define="
-		field python:context.Schema()['Remarks'];
-		errors python:{};">
-		<p style="margin-top:2em;"/>
-		 <metal:widget use-macro="python:context.widget('Remarks', mode='edit')" />
-	</tal:remarks>
+      <!-- Listing Table -->
+      <div id="folderlisting-main-table"
+           tal:content="structure view/contents_table"/>
 
-</metal:content-core>
+      <tal:remarks define="field python:context.Schema()['Remarks'];
+                           errors python:{};">
+        <p style="margin-top:2em;"/>
+        <metal:widget use-macro="python:context.widget('Remarks', mode='edit')" />
+      </tal:remarks>
```

Comment:
I think it would be better if we remove this remarks field from this "Add Analyses" view.